File: app/CentralController/EEGDataPersistence/api/message/MessageKeyEnum.java
package com.coreplantform.EEGDataPersisence.api.message;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public enum MessageKeyEnum {
    COMMAND_CONTROL("command_control"),

    GROUP_1.DATA("group_1.data"),//脑电数据传输

    GROUP_2.DATA("group_2.data"),//脑电数据传输
    GROUP_3.DATA("group_3.data");//脑电数据传输
    private final String messageKey;


}

File: app/Stimulator/Stimulator/facade/message/MessageKeyEnum.py
from enum import Enum


class MessageKeyEnum(Enum):
    """
    定义本系统发送/接收消息所需message_key
    枚举对象为系统所引对象，枚举值为message_key，枚举值对应于配置文件中的message字段中的key
    """
    EXTERNAL_TRIGGER = "external_trigger"


